adapters/jwplayer/utils_test.go

```diff
@@ -0,0 +1,239 @@
+package jwplayer
+
+import (
+	"encoding/json"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/stretchr/testify/assert"
+	"testing"
+	"text/template"
+)
+
+func TestParseExtraInfo(t *testing.T) {
+	extraInfo := ParseExtraInfo("{\"targeting_endpoint\": \"targetingUrl\"}")
+	assert.Equal(t, "targetingUrl", extraInfo.TargetingEndpoint)
+
+	defaultTargetingUrl := "https://content-targeting-api.longtailvideo.com/property/{{.SiteId}}/content_segments?content_url=%{{.MediaUrl}}&title={{.Title}}&description={{.Description}}"
```

Comment:
same here -- and does the test actually hit the endpoint? should we have a mock response instead?

Reply:
none of the tests make real requests. This test is just checking that the right targeting url is obtained. The tests that check requests make a mock server.